lms/djangoapps/discussion_api/views.py

```diff
@@ -158,7 +159,9 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
             "unanswered" for question threads with no marked answer. Only one
             can be selected.
 
-        The topic_id, text_search, and following parameters are mutually
+        * inline_discussion_id: ID of an inline discussion for any module
```

Comment:
@catong We aren't bought into the "inline discussion" terminology yet.  We may just use "discussion_id" here to accommodate querying by any discussion identifier.

I don't remember who from the documentation team had reviewed the initial wording in the discussion API last year.  If you'd like to go through the entire file, please do so.  We can make any documentation fixes as part of a separate PR as well, so we don't need to hold up this work (and the subsequent integration testing with the mobile team).

I agree that the "any module" phrase can be updated/removed.